src/model/FormGenerator.js

```diff
@@ -20,7 +20,7 @@ export default class FormGenerator {
 
     const [formQuestions, form] = FormGenerator._constructFormQuestions(defaultFormData, intl);
 
-    return [formQuestions, { root: form }];
+    return [formQuestions, form];
```

Comment:
!!! I am afraid this is a mistake !!!!

This is not a compatible change ... does it really work like that without the need to change the behavior of other components?


Look at FormGenerator.js:105:
```
    // sort by property
    JsonLdObjectUtils.orderPreservingToplogicalSort(formQuestions, Constants.HAS_PRECEDING_QUESTION);

    return [formQuestions, { root: form }];
  }
```

Why should default form have different structure than not-default one ?